src/librc/librc.c

```diff
@@ -302,6 +304,8 @@ detect_container(const char *systype RC_UNUSED)
 		return RC_SYS_RKT;
 	else if (file_regex("/proc/1/environ", "container=systemd-nspawn"))
 		return RC_SYS_SYSTEMD_NSPAWN;
+	else if (exists("/run/.containerenv"))
```

Comment:
*bsds don't have /run, podman docs suggest that /var/run/.containerenv will be used on freebsd containers: https://docs.podman.io/en/latest/markdown/podman-run.1.html

should we add both here? (and maybe under an ifdef)

Reply:
The check exists in an `#ifdef __linux__`. I could add check for `/var/run/.containerenv` under the pre-existing `#ifdef __FreeBSD__` in the function (Podman only mentions FreeBSD).

Reply:
oh, yes. thank you